nodejs/eks/cluster.ts

```diff
@@ -483,14 +485,55 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
             return mappingYaml;
         });
     }
-    const eksNodeAccess = new k8s.core.v1.ConfigMap(`${name}-nodeAccess`, {
-        apiVersion: "v1",
-        metadata: {
-            name: `aws-auth`,
-            namespace: "kube-system",
-        },
-        data: nodeAccessData,
-    }, { parent: parent, provider: provider });
+    /*
+    * We must kubectl apply the aws-auth configmap since the creation of an
+    * AWS managed node group automatically triggers AWS to create the aws-auth
+    * configmap for us.
+    *
+    * For managed nodegroups:
+    *   To properly create aws-auth, we need to kubectl apply the configmap
+    *   instead of kubectl create it, if not it will fail with an existing
+    *   resource error message.
+    *
+    * For self-managed nodegroups:
+    *   We continue to always create the aws-auth configmap.
+    *
+    * As a work around, we'll kubectl apply to satisfy both node group cases.
+    *
+    * TODO(metral)
+    * When https://git.io/JeXom is resolved, we can move away from kubectl
+    * shell out.
```

Comment:
Nit - in most other repos we've used the style `TODO[pulumi/pulumi-kubernetes#264]: Note about the issue.`

Reply:
Done